ruby/vampires.rb

```diff
@@ -0,0 +1,56 @@
+puts "How many employees you will be processed?"
+num_empl = gets.chomp.to_i 
+
+
+i=0
+until i == num_empl
```

Comment:
Look into the times loop for this type of iteration; 
ruby-doc.org/core-2.2.0/Integer.html#method-i-times